### ISSUE TITLE
[Feature Request] Local storage

### ISSUE THREAD

**Issue by sparcut** [ecmadao/chrome-utils#4]
Need option to use local store.

At the moment you can only use sync storage.

**Comment by ecmadao** [comment]
Yes, it's a good idea, and I also have some plan for it. But chrome-utils is mainly developed for chrome extensions, so I only use chrome extension raw api: `chrome.storage`. As for other storage ways, such as local storage, session storage, indexed db, web sql, I think they should be separated as another web frontend storage package.
Any way, thanks for your suggestion. Maybe I can work for the new package some times later.

**Comment by sparcut** [comment]
Sorry, I feel you've misunderstood what I ment by local store.

I'm not referring to the `localStorage`, rather the chrome local storage: `chrome.storage.local`. I use it for storing things that I don't want synced e.g. state, whereas I use sync for things like settings.

https://developer.chrome.com/apps/storage

**Comment by ecmadao** [comment]
Sounds great! I think this feature can be added to the module. But these days I have some trivia to deal with, maybe I can start to work on it a few days later.
